src/components/K8s.vue

```diff
@@ -51,8 +63,56 @@ export default {
   computed: {
     cannotReset: function() {
       return (this.state !== K8s.State.STARTED && this.state !== K8s.State.READY);
+    },
+
+    memoryValueIsValid: function() {
+      let value = this.settings.kubernetes.memoryInGB;
+      let numericValue;
+      if (typeof(value) !== "number") {
+        numericValue = parseFloat(value, 10);
+        if (isNaN(numericValue)) {
+          return false;
+        }
+      }
+      if (numericValue < 2 ||
+          (numericValue > this.availMemoryInGB && this.availMemoryInGB)) {
+        return false;
+      }
+      return true;
+    },
+
+    numCPUsValueIsValid: function() {
+      let value = this.settings.kubernetes.numberCPUs;
+      let numericValue;
+      if (typeof (value) !== "number") {
+        numericValue = parseInt(value, 10);
+        if (isNaN(numericValue)) {
+          return false;
+        }
+      } else {
+        numericValue = value;
+      }
+      if (numericValue < 2 ||
+        (numericValue > this.availNumCPUs && this.availNumCPUs)) {
+        return false;
+      }
+      return true;
     }
-  }, 
+  },
+
+  created() {
+    this.debouncedActOnUpdateMemory = debounce(this.actOnUpdatedMemory, 1000);
+    this.debouncedActOnUpdateCPUs = debounce(this.actOnUpdatedCPUs, 1000);
+    const totalMemInGB = os.totalmem() / 2**30;
+    const reservedMemoryInGB = 6; // should be higher?
```

Comment:
Dumb thoughts:
- Should we calculate `availableMemoryInGB` inside `memoryValueIsValid` instead, in case the user somehow hotplugs memory? (More realistically, we're running inside a VM, and the memory limits change.)
- If yes, would we have to hook up a listener to update the UI? Probably not, given that the `os` module doesn't have a good way to do so.

Reply:
Let's mark that out of scope for the time being